### DISCUSSION TITLE
Dashboard for CometBFT metrics

### DISCUSSION THREAD

**Discussion by begmaroman** [cosmos/community#61] [Chain Builders (Cosmos SDK)]
Dear CometBFT Team,

I hope this message finds you well. I am a user of CometBFT and am currently exploring ways to enhance the monitoring and visualization of Prometheus metrics generated by CometBFT.

I understand that you may have Grafana or Datadog dashboards to visualize these metrics, and I was wondering if it would be possible for you to share those dashboards or any relevant configuration settings. Having access to such dashboards would greatly assist in optimizing our monitoring setup and improving the overall understanding of CometBFT performance.

I appreciate your time and efforts in developing CometBFT, and I believe that sharing these dashboards would be beneficial to the wider community of users.

Thank you for your consideration. Looking forward to your positive response.

**Comment by andynog** [comment]
As far as I know, we don't maintain a dashboard online, but if you're asking about setting up some grafana to monitor some CometBFT metrics, you can maybe leverage the logic in the CometBFT's QA infra repo -> https://github.com/cometbft/qa-infra/tree/main/grafana

**Comment by adizere** [comment]
> I understand that you may have Grafana or Datadog dashboards to visualize these metrics, and I was wondering if it would be possible for you to share those dashboards or any relevant configuration settings. 

Is there a particular network that would interest you? For instance, Cosmos Hub, or Osmosis? CometBFT deployments and Grafana dashboard tend to differ by network, though there are many overlapping concerns.

The [link](https://github.com/cometbft/qa-infra/tree/main/grafana) that Andy shared above shows how the core CometBFT team assesses performance of Comet in our QA tests we do for every major release. Was that useful?
